libraries/rush-sdk/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./node_modules/local-node-rig/profiles/default/tsconfig-base.json",
   "compilerOptions": {
-    "outDir": "lib-shim",
+    "outDir": "lib-commonjs",
```

Comment:
This should go in a folder called something like `lib-shim-esnext` and provide ES-style modules to Webpack.

Reply:
By following rush-lib's fashion, configuring `additionalModuleKindsToEmit: [ { moduleKind: 'esnext', { outFolderName: 'lib-esnext ' } ]`, and update rush-sdk's webpack.config.js to use `lib-esnext` folder as entries.

Reply:
Unfornunately, using `lib-esnext` as entries don't work well with `rush-sdk`. In scenario 2, rush-sdk uses `module.parent` to read the parent module filename. This doesn't work in ESM code... which breaks the test cases for rush plugins in this repo. So, I reverted the webpack.config.js to use commonjs code as entries.